plugin/lsp_settings.vim

```diff
@@ -97,7 +97,7 @@ function! s:vimlsp_settings_suggest() abort
   if empty(s:vimlsp_installer())
     return
   endif
-  echomsg printf('If you want to enable Language Server, please do :LspInstallServer')
+  echomsg printf('%s', 'If you want to enable Language Server, please do :LspInstallServer')
```

Comment:
Thank you. Please do

```
echomsg 'If you want to enable Language Server, please do :LspInstallServer'
```

Reply:
Thanks for your review. I updated my pull request.